recipes/arpack/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+else
+  SO_EXT='.so'
+fi
+
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_SHARED_LIBS=ON \
```

Comment:
That has been the status quo is trying to provide both dynamic and static libraries so people can chose what is appropriate for their use case.